plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryIntegrationSmokeTest.java

```diff
@@ -47,6 +47,39 @@ protected QueryRunner createQueryRunner()
                 ImmutableMap.of());
     }
 
+    @Test
+    public void testCreateSchema()
+    {
+        String schemaName = "test_create_schema";
+
+        assertUpdate("DROP SCHEMA IF EXISTS " + schemaName);
+
+        assertUpdate("CREATE SCHEMA " + schemaName);
+        assertUpdate("CREATE SCHEMA IF NOT EXISTS " + schemaName);
+
+        assertQueryFails(
+                "CREATE SCHEMA " + schemaName,
+                format("\\Qline 1:1: Schema 'bigquery.%s' already exists\\E", schemaName));
+
+        assertUpdate("DROP SCHEMA " + schemaName);
+    }
+
+    @Test
```

Comment:
Please add a test when you try to drop schema in case when case insensitive mapping is enabled.
Test a case when only single schema matches `DROP` request (it should be allowed).
And a case when there are multiple matching schemas. In that case `DROP` should raise an exception.

Reply:
perhaps instead of adding expicit tests here we could leverage `io.trino.testing.BaseConnectorSmokeTest#testCreateSchema` or `io.trino.testing.BaseConnectorTest (AbstractTestDistributedQueries)#testCreateSchema`